src/plot/xy-plot.js

```diff
@@ -56,6 +56,8 @@ const DEFAULT_MARGINS = {
   bottom: 40
 };
 
+const DEFAULT_PIXEL_RATIO = window && window.devicePixelRatio || 1;
```

Comment:
hmm it'd probably be better if this was a default prop in the canvas-wrapper component? sorry to flip around on this

Reply:
No problem! I've changed it.